lib/sisimai/bite/json/sendgrid.rb

```diff
@@ -11,65 +11,56 @@ def description; return 'SendGrid(JSON): http://sendgrid.com/'; end
 
       # @abstract Adapt SendGrid bounce object for Sisimai::Message format
       # @param        [Hash] argvs  bounce object(JSON) retrieved from SendGrid API
-      # @return       [Hash, Nil]   Bounce data list and message/rfc822 part
-      #                             or Undef if it failed to parse or the
+      # @return       [Hash, Nil]   Bounce data list and message/rfc822 part or
+      #                             nil if it failed to parse or the
       #                             arguments are missing
       # @since v4.20.0
       def adapt(argvs)
         return nil unless argvs.is_a? Hash
         return nil unless argvs.keys.size > 0
         return nil unless argvs.key?('email')
+        return nil unless Sisimai::RFC5322.is_emailaddress(argvs['email'])
 
         dscontents = [Sisimai::Bite.DELIVERYSTATUS]
-        rfc822head = {}     # (Hash) Check flags for headers in RFC822 part
-        recipients = 0      # (Integer) The number of 'Final-Recipient' header
         v = dscontents[-1]
 
         require 'sisimai/string'
         require 'sisimai/address'
 
-        if Sisimai::RFC5322.is_emailaddress(argvs['email'])
-          #   {
-          #       "status": "4.0.0",
-          #       "created": "2011-09-16 22:02:19",
-          #       "reason": "Unable to resolve MX host sendgrid.ne",
-          #       "email": "esting@sendgrid.ne"
-          #   },
-          recipients += 1
-          v['recipient'] = argvs['email']
-          v['date'] = argvs['created'] || ''
+        #   {
+        #       "status": "4.0.0",
+        #       "created": "2011-09-16 22:02:19",
+        #       "reason": "Unable to resolve MX host sendgrid.ne",
+        #       "email": "esting@sendgrid.ne"
+        #   },
+        v['recipient'] = argvs['email']
+        v['date'] = argvs['created'] || ''
 
-          statuscode = argvs['status']  || ''
-          diagnostic = Sisimai::String.sweep(argvs['reason']) || ''
+        statuscode = argvs['status']  || ''
```

Comment:
__[rubocop]__
* Layout/SpaceAroundOperators: Operator `||` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->